### PR TITLE
Add new filters to the CDR Exports

### PR DESCRIPTION
**Pull request by Skumring**
Add additional CDR export filters: customer_auth_external_type_eq, customer_auth_external_type_not_eq, customer_auth_external_id_in (supports array), src_country_iso_in (supports array), dst_country_iso_in (supports array)

filters
cdr_export_show

### CODE REVIEW

**Comment by dmitry-sinina** [comment]
Is is possible to add "Copy feature". To create new CDR export based on values from previous one?

**Comment by dmitry-sinina** [comment]
There is copy button in every object show page, for example customer auth:

image

But by some reasons there is no such button in CDR export. It would be quite useful to have it to create same CDR export where only date range is different.

**Comment by Skumring** [comment]
> There is copy button in every object show page, for example customer auth:
> 
> image
> But by some reasons there is no such button in CDR export. It would be quite useful to have it to create same CDR export where only date range is different.

Date range - means created_at&updated_at timestamps of CdrExport or you mean filters: `time_start_gteq` & `time_start_lteq` & `time_start_lt` inside json attribute `filters` in CdrExport

**Comment by gigorok** [comment]
> Is is possible to add "Copy feature". To create new CDR export based on values from previous one? Same as we have for other objects.

@dmitry-sinina could you create a new issue? "Copy feature" will be implemented in another scope of improvements

File: app/admin/cdr/cdr_exports.rb [hunk 1]
```diff
@@ -225,6 +271,26 @@ def build_new_resource
                input_html: { class: 'chosen' },
                required: false
       ff.input :term_gw_external_id_eq, required: false
+      ff.input :customer_auth_external_type_eq, required: false
+      ff.input :customer_auth_external_type_not_eq, required: false
+      ff.input :customer_auth_external_id_in,
+               as: :select,
+               multiple: true,
+               collection: CustomersAuth.order(:name).pluck(:name, :external_id),
```
Comment (gigorok) [inline app/admin/cdr/cdr_exports.rb:279]: could it be implemented using Ajax? possible big amount of customers auth records

**Pull request merged**